website/docs/d/datasource_google_service_account_key.html.markdown

```diff
@@ -14,27 +14,27 @@ Get service account public key. For more information, see [the official document
 ## Example Usage
 
 ```hcl
-data "google_service_account" "myaccount" {
-  account_id = "myaccount"
+resource "google_service_account" "myaccount" {
+  account_id = "dev-foo-account"
 }
 
-data "google_service_account_key" "mykey" {
-  service_account_id = "${data.google_service_account.myaccount.name}"
-  public_key_type = "TYPE_X509_PEM_FILE"
+resource "google_service_account_key" "mykey" {
+  service_account_id = "${google_service_account.myaccount.name}"
 }
 
-output "mykey_public_key" {
-  value = "${data.google_service_account_key.mykey.public_key}"
+data "google_service_account_key" "mykey" {
+  name = "${google_service_account_key.mykey.name}"
```

Comment:
bleh this example is super contrived, although I don't think there are any other resources that output keys that you could input into the data source so I don't have any other ideas.

Reply:
yeah...yeah.